docs/sites.yml

```
@@ -4516,3 +4516,14 @@
   built_by: Nick Taylor
   built_by_url: https://www.iamdeveloper.com
   featured: false
+- title: AquaGruppen Vattenfilter
+  url: https://aquagruppen.se
+  main_url: https://aquagruppen.se/
+  description: >
+    Vattenfilter och vattenrening (Water filter and water treatment products in Sweden)
```

```suggestion
    Water filter and water treatment products in Sweden
```

Let's just default to English here, if we can!